react/introduction/setting_up_a_react_environment.md

```diff
@@ -0,0 +1,145 @@
+### Introduction
+
+You now know _what_ React is, you might even have an idea of _where_ to use it, it's time to start figuring out _how_. 
+
+This lesson is going to cover starting a new React project on your own machine, as well as some useful tools to help you along the way. We'll also explain some of the problems that may arise (and how to avoid them).
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+* How React projects can be created locally.
+* Describing the difficulties when starting a React project without the appropriate tools.
+* Listing some of the popular tools and methods for scaffolding a new React project.
+* Explaining what Create React App is.
+* Setting up your workflow to develop efficiently within your new project.
```

Comment:
Lesson Overviews should be much more general rather than specific. Here's our wiki page on the topic: https://github.com/TheOdinProject/curriculum/wiki/Lesson-Overviews-vs-Knowledge-Checks

Some suggestions:
```suggestion
* How React projects can be created.
* How to use Create React App.
* What are React Developer Tools.
```

Reply:
Have reworded the current overview and added the suggestions. Let me know your thoughts on the new ones if you'd like further changes to be made.